versioned_docs/version-8.6/apis-tools/camunda-api-rest/sidebar-schema.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   "Camunda 8 API (REST)": [
     "apis-tools/camunda-api-rest/camunda-api-rest-overview",
     "apis-tools/camunda-api-rest/camunda-api-rest-authentication",
+    "apis-tools/camunda-api-rest/camunda-8-api-tutorial",
```

Comment:
❓ Do we need to update a versioned sidebar for this as well? I cannot see this in 8.6 and 8.7 in the navigation.